x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout/helpers.tsx

```diff
@@ -31,6 +31,11 @@ const actionNames: {
     completedText: 'force unenrolled',
     cancelledText: 'force unenrollment',
   },
+  AUTOMATICAL_FORCE_UNENROLL: {
+    inProgressText: 'Automatical unenrolling',
+    completedText: 'automatically unenrolled',
+    cancelledText: 'automatical unenrollment',
+  },
```

Comment:
```suggestion
  AUTOMATIC_FORCE_UNENROLL: {
    inProgressText: 'Automatic unenrolling',
    completedText: 'automatically unenrolled',
    cancelledText: 'automatic unenrollment',
  },
```

Just fixing some wording here